docs/howto/manage-connections.rst

```diff
@@ -151,6 +151,52 @@ Scopes (comma separated)
         issue `AIRFLOW-2522
         <https://issues.apache.org/jira/browse/AIRFLOW-2522>`_.
 
+Amazon Web Services
+~~~~~~~~~~~~~~~~~~~
+
+The Amazon Web Services connection type enables the :ref:`AWS Integrations
+<AWS>`.
+
+Authenticating to AWS
+'''''''''''''''''''''
+
+Authentication may be performed using any of the `boto3 options <https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html#configuring-credentials>`_. Alternatively, one can pass credentials in as a Connection initialisation parameter.
+
+Default Connection IDs
+''''''''''''''''''''''
+
+The default connection ID is ``aws_default``.
+
+Configuring the Connection
+''''''''''''''''''''''''''
+
+Login (optional)
+    Specify the AWS access key ID.
+
+Password (optional)
+    Specify the AWS secret access key.
+
+Extra (optional)
+    Specify the extra parameters (as json dictionary) that can be used in ssh
+    connection. The following parameters are supported:
+
+    * **aws_access_key_id**: AWS access key ID of the connection. To use this, the `aws_secret_access_key` parameter must also be passed in.
+    * **aws_secret_access_key**: AWS secret access key of the connection
```

Comment:
I would rather we didn't document the key_id and access_key in extra, as those really belong in the login and password.